CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for contributing!
+
+## Authoring
+
+Code should be well documented and tested. Each loader must:
+
+* Document for users via its own README
+* Document for developers via code comments
+* [Type-check via jsdoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)
+* Cover at least 90% of code between unit and end-to-end tests
+  * unit tests for branching and especially failure cases
+  * end-to-end for [happy-paths](https://en.wikipedia.org/wiki/Happy_path)
+  * You can run tests locally via `npm run test --workspace=packages/LOADER_NAME`
```

Comment:
it's work ? I never do that because our test command is on root

Reply:
Yes, you must run it from root. But yes it works :)